src/main/java/no/digipost/http/client/Validation.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) Posten Norge AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package no.digipost.http.client;
+
+public class Validation {
+
+    public static int NOrGreater(int value, int threshold, String nameOfValue) {
+        if (value < threshold) {
```

Comment:
Sjekker bare "større enn", godtar ikke dersom `value == threshold` selv om metoden sier den skal gjøre det.
`DigipostHttpClientMillisecondTimeouts` godtar nå ikke 0-verdi (uendelig) som timeout, noe den bør gjøre for debugging.